foreshadow/utils/validation.py

```diff
@@ -11,10 +11,47 @@
 PipelineStep = {"NAME": 0, "CLASS": 1, "COLS": 2}
 
 
+def check_series(input_data):
```

Comment:
Can you explain to me the use case. I have some in mind but I assume you've already hit one (for my own knowledge).

Reply:
Just to allow me to perform series operations on an input knowing that it's a series

Reply:
cool!

Reply:
DOcument that this is useful for computations on single columns such as metrics.

Reply:
Done